src/backend/gl/src/window/wgl.rs

```diff
@@ -186,12 +183,16 @@ impl Instance {
         }
     }
 
-    #[cfg(feature = "winit")]
-    pub fn create_surface(&self, window: &winit::window::Window) -> Surface {
-        use winit::platform::windows::WindowExtWindows;
-
-        let hwnd = window.hwnd();
-        self.create_surface_from_hwnd(hwnd as *mut _)
+    #[cfg(windows)]
```

Comment:
Not quite sure about this, or if the entire wgl feature should be cfg(target_os = windows), or whether the unreachable should instead be something like panic!("WGL is not supported for this target")?

Reply:
latter sounds good to me. @msiglreith ?